test/test_poolmanager.py

```diff
@@ -276,6 +276,16 @@ def test_http_connection_from_host_case_insensitive(self):
         self.assertTrue(pool is other_pool)
         self.assertTrue(all(isinstance(key, PoolKey) for key in p.pools.keys()))
 
+    def test_assert_hostname_and_fingerprint_flag(self):
+        """Assert scheme case is ignored when getting the https key class."""
```

Comment:
This docstring isn't quite right, do you mind rewording it to explain what is going on in this test?